include/zephyr/drivers/clock_control/clock_control_tcc_ccu.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2024 Hounjoung Rim <hounjoung@tsnlab.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+#ifndef CLOCK_CONTROL_TCC_CCU_HEADER
+#define CLOCK_CONTROL_TCC_CCU_HEADER
+
+/*
+ * DEFINITIONS
+ *
+ */
+
+#define CKC_MHZ(x) (uint32_t)((x) * 1000000)
+
+/*
+ * FUNCTION PROTOTYPES
+ *
+ */
+
+void vcp_clock_init(void);
+
+long clock_set_pll_rate(signed long id, uint32_t rate);
+
+uint32_t clock_get_pll_rate(signed long id);
```

Comment:
None of these are documented, doesn't look to be using the zephyr driver class properly @nordic-krch is this acceptable?